### PR TITLE
PFCF-202: The users are notified if they have not entered the required fields in each page and are prevented from navigating to the next page.

### PR DESCRIPTION
**Pull request by marzmehr**

### CODE REVIEW

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
The entry validation appears to work, however when something is not filled out the user is taken to the top of the page, rather than the first instance of a question that has not been answered.  This is a little confusing, see screenshots.

Would it be possible to move to the first unanswered question to make it clear why the screen jumped?

image

image

**Comment by marzmehr** [comment]
> The entry validation appears to work, however when something is not filled out the user is taken to the top of the page, rather than the first instance of a question that has not been answered. This is a little confusing, see screenshots.
> 
> Would it be possible to move to the first unanswered question to make it clear why the screen jumped?
> 
> image
> 
> image

I'll look into it @WadeBarnes . Thank you for the feedback.

File: fpo-api/fpo_api/urls.py [hunk 1]
```diff
@@ -20,5 +20,6 @@
     path("health/", views.health)
 ]
 
-if (os.getenv("FRONT_END", "prod") == "dev"): 
+if (os.getenv("FRONT_END", "prod") == "dev"):
+    urlpatterns.append(path("family-law-act/family-law-act/api/v1/", include("api.urls")))
```
Comment (WadeBarnes) [inline fpo-api/fpo_api/urls.py:24]: Is the duplicated path required for `dev`?

**Comment by WadeBarnes** [comment]
The issue reported above is fixed, however, now when you navigate from one page to another you are not returned to the top of a new page when it fills more than a single screen.

For example, fill out the first page of the PO, and click next:

image

You end up being placed part way down on the next page:

image

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
See above.

File: fpo-api/fpo_api/urls.py (hunk 1 above)
Comment (marzmehr) [reply fpo-api/fpo_api/urls.py:24]: Yes,
For dev environment: I run the API and DB in docker, I run the front-end in dev on port 8084, 
I have to use this path (duplicate:  'family-law-act/family-law-act/api/v1/' ) in order to hit the API.

**Comment by marzmehr** [comment]
Closing, since the changes are now in PFCF-187

**Pull request closed without merging**